config/locales/en.yml

```diff
@@ -65,6 +66,48 @@ en:
           update:
             invalid: There was a problem updating this answer
             success: Answer successfully updated
+        census:
+          create:
+            error: There was an error importing or generating the census.
+            success:
+              generate: Successfully generated the census
+              import: Successfully imported %{count} items (%{errors} errors)
+          destroy:
+            confirm: Delete all the census can not be undone. Are you sure you want
+              to continue?
+            title: Delete all census data
+          destroy_all:
+            success: All census data have been deleted
+          generate:
+            info: You can generate the credentials for all the participants with the
```

Comment:
This should be "must"

Reply:
Yes, indeed. Done at 3133c60758802df4b42c359168d6253ead23a4a5